pkg/cmd/util/clientcmd/factory.go

```diff
@@ -489,6 +489,11 @@ func (c *clientCache) ClientConfigForVersion(version string) (*kclient.Config, e
 	client.SetOpenShiftDefaults(&config)
 	c.configs[version] = &config
 
+	// `version` does not necessarily equal `config.Version`.  However, we know that we call this method again with
+	// `config.Version`, we should get the the config we've just built.
+	config = config
```

Comment:
@liggitt is there an easy way to make verify happy?

```
pkg/cmd/util/clientcmd/factory.go:494: self-assignment of config to config
```

My intent here is to create another copy to ensure distinct references in the cache.